src/scss/site/_carbon-overrides.scss

```diff
@@ -46,7 +46,7 @@
 }
 
 .content {
-  ul:not(.bx--tabs__nav),
+  ul:not(.bx--tabs__nav):not(.card--list--container),
```

Comment:
This removes `padding-left` for the `ul` element containing the cards. Removing  `padding-left` makes the cards align with the header on all viewport sizes